Dockerfiles/Dockerfile.app

```diff
@@ -9,4 +9,6 @@ RUN pip install -r requirements.txt
 # copy project
 COPY . .
 
+RUN flask init-es
```

Comment:
I don't think that this would work? We build the image before the ES is started.
Maybe we can move it to entrypoint?